From: ski <ski@opencascade.com>
Date: Tue, 26 Apr 2016 13:02:32 +0000 (+0300)
Subject: 0027435: Compilation error on Linux platfrom in Debug mode with enabled OCCT_DEBUG... 
X-Git-Tag: V7_0_winwerth~75
X-Git-Url: http://git.dev.opencascade.org/gitweb/?p=occt.git;a=commitdiff_plain;h=4b5857d330d381ea52951f7b64336b455187a948

0027435: Compilation error on Linux platfrom in Debug mode with enabled OCCT_DEBUG definition

Error was fixed.
---

diff --git a/src/MAT2d/MAT2d_Tool2d.cxx b/src/MAT2d/MAT2d_Tool2d.cxx
index 0175199..8b4324d 100644
--- a/src/MAT2d/MAT2d_Tool2d.cxx
+++ b/src/MAT2d/MAT2d_Tool2d.cxx
@@ -1078,7 +1078,7 @@ void MAT2d_Tool2d::Dump(const Standard_Integer bisector,
   if(bisector == -1) return;
   if(bisector > theNumberOfBisectors) return;
 
-  Handle(Geom2d_Curve) thebisector = GeomBis(bisector).Value();
+  Handle(Geom2d_Curve) thebisector = (Handle(Geom2d_Curve)) GeomBis(bisector).Value();
 
   MAT2d_DrawCurve(thebisector,3);
 
